src/database/migrations/2022_05_17_140121_create_comments_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateCommentsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('comments', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('movie_id');
+            $table->unsignedInteger('user_id');
+            $table->foreign('movie_id')->references('id')->on('movies')->onDelete('cascade');
+            $table->foreign('user_id')->references('id')->on('users')->onDelete('cascade');
```

Comment:
$table->unsignedBigInteger('movie_id');
  $table->foreign('movie_id')->references('id')->on('movies')->onDelete('cascade');

ovaj deo mozes i ovako
$table->foreignId('movie_id')->constrained('movies')->cascadeOnDelete();

Reply:
I za sve ostale isto tako umesto 2 linije mozes to u jednoj